core/services/relay/evm/functions/logpoller_wrapper.go

```diff
@@ -151,7 +151,7 @@ func (l *logPollerWrapper) LatestEvents() ([]evmRelayTypes.OracleRequest, []evmR
 	if l.proposedCoordinator != (common.Address{}) && l.activeCoordinator != l.proposedCoordinator {
 		coordinators = append(coordinators, l.proposedCoordinator)
 	}
-	latest, err := l.logPoller.LatestBlock()
+	latest, err := l.logPoller.LatestBlock(context.Background())
```

Comment:
Think without calling cancel somewhere this is just a placeholder really. Maybe we want to do something like 
```suggestion
        ctx, cancel := context.WithCancel(context.Background())
        defer cancel()
	latest, err := l.logPoller.LatestBlock(ctx)
```

Reply:
Fair point, I'll make this change throughout.

Reply:
Noticed we're still passing the `context.Background()` here still. We can probably move up the context you're creating on line 174 up here and use the same context throughout

Reply:
Just to reaffirm what Jordan said so I'm not giving mixed signals with my original comment. Think we should look up the call stack to see if there's a context we can pass down to this method.

Reply:
Yeah, we shouldn't create a context here, we should pass down the context from the callers.

Reply:
Changed to use context from callers